ShopperInsights/src/main/java/com/braintreepayments/api/shopperinsights/v2/internal/CustomerSessionRequestBuilder.kt

```diff
@@ -0,0 +1,57 @@
+package com.braintreepayments.api.shopperinsights.v2.internal
+
+import com.braintreepayments.api.core.ExperimentalBetaApi
+import com.braintreepayments.api.shopperinsights.v2.CustomerSessionRequest
+import org.json.JSONArray
+import org.json.JSONObject
+
+/**
+ * Builder for creating the JSON request objects for the Shopper Insights v2 APIs.
+ */
+@ExperimentalBetaApi
+class CustomerSessionRequestBuilder {
+
+    internal data class JsonRequestObjects(
+        val customer: JSONObject,
```

Comment:
Not at all a blocker, just curious. Is there a reason why we use `Json` when Kotlin seems to prefer `JSON`? Not sure if that is a Java holdover or if the language follows a bit of an anti-pattern. 🙈

Reply:
That's a great question! In general, we should be following pascal case and camel case, as they're the preferred, modern naming conventions.

The Json classes that Android includes is from the `org.json` library, which is an extremely old Java Json library. It was created when Java naming conventions were less formalized.